sdk/attestation/attestation/test/utils/base64.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * Encodes a string in base64 format.
+ * @param value the string to encode
+ */
+export function encodeString(value: string): string {
+  return Buffer.from(value).toString("base64");
+}
+
+/**
+ * Encodes a byte array in base64 format.
+ * @param value the Uint8Aray to encode
+ */
+export function encodeByteArray(value: Uint8Array): string {
+  // Buffer.from accepts <ArrayBuffer> | <SharedArrayBuffer>-- the TypeScript definition is off here
+  // https://nodejs.org/api/buffer.html#buffer_class_method_buffer_from_arraybuffer_byteoffset_length
+  const bufferValue = value instanceof Buffer ? value : Buffer.from(value.buffer as ArrayBuffer);
```

Comment:
Nit: Not sure if this is a guideline but I think it's a good idea to stay away from `instanceof` checks if an alternative exists when possible. This is because it's possible in JS to load multiple copies of the same dependency which could cause instanceof to return false when you'd expect true.

In this case since `Buffer.isBuffer` exists can we use that?
```suggestion
  const bufferValue = Buffer.isBuffer(value) ? value : Buffer.from(value.buffer as ArrayBuffer);
```